### ISSUE TITLE
TFLite initializes Coral device but still runs inference on CPU

### ISSUE THREAD

**Issue by damjandakic93** [tensorflow/tensorflow#70803]
### Issue type

Bug

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

source

### TensorFlow version

2.13.1

### Custom code

Yes

### Current behavior?

I have a model which I wish to run on RPi-4 with Coral USB device connected to it. I've set the context manager verbosity to max. I can see that the library is communicating with the device, however, in the end, it still runs inference on CPU. I have verified this by measuring time and checking the CPU utilization.

Notice: "INFO: Created TensorFlow Lite XNNPACK delegate for CPU." in the middle of the logs.

### Standalone code to reproduce the issue

```shell
#include <memory>
#include <iostream>
#include "tensorflow/lite/core/interpreter.h"
#include "tensorflow/lite/core/model_builder.h"
#include "tensorflow/lite/kernels/register.h"
#include "tflite/public/edgetpu.h"
#include "flatbuffers/flatbuffers.h"
#include <chrono>

using namespace std::chrono;

std::unique_ptr<tflite::Interpreter> BuildEdgeTpuInterpreter(
    const tflite::FlatBufferModel &model,
    edgetpu::EdgeTpuContext *edgetpu_context)
{
    tflite::ops::builtin::BuiltinOpResolver resolver;
    resolver.AddCustom(edgetpu::kCustomOp, edgetpu::RegisterCustomOp());
    std::unique_ptr<tflite::Interpreter> interpreter;
    if (tflite::InterpreterBuilder(model, resolver)(&interpreter) != kTfLiteOk)
    {
        std::cerr << "Failed to build interpreter." << std::endl;
    }
    // Bind given context with interpreter.
    interpreter->SetExternalContext(kTfLiteEdgeTpuContext, edgetpu_context);
    interpreter->SetNumThreads(-1);
    if (interpreter->AllocateTensors() != kTfLiteOk)
    {
        std::cerr << "Failed to allocate tensors." << std::endl;
    }
    return interpreter;
}

std::unique_ptr<tflite::Interpreter> Initialize()
{
    const std::string model_path = "converted_model.tflite";
    std::unique_ptr<tflite::FlatBufferModel> model =
        tflite::FlatBufferModel::BuildFromFile(model_path.c_str());

    edgetpu::EdgeTpuManager::GetSingleton()->SetVerbosity(10);

    std::shared_ptr<edgetpu::EdgeTpuContext> edgetpu_context =
        edgetpu::EdgeTpuManager::GetSingleton()->OpenDevice();

    std::unique_ptr<tflite::Interpreter> model_interpreter =
        BuildEdgeTpuInterpreter(*model, edgetpu_context.get());

    return model_interpreter;
}

int main()
{
    std::cout << "Hello world!" << std::endl;
    std::unique_ptr<tflite::Interpreter> interpreter = Initialize();
    // Fill `input`

    auto start = high_resolution_clock::now();

    interpreter->Invoke();

    auto stop = high_resolution_clock::now();
    auto duration = duration_cast<microseconds>(stop - start);
    std::cout << duration.count() << std::endl;
    float *output = interpreter->typed_output_tensor<float>(0);
    std::cout << output[0] << std::endl;

    return 0;
}
```

### Relevant log output

```shell
Hello world!
I tflite/edgetpu_manager_direct.cc:453] No matching device is already opened for shared ownership.
I driver/driver_factory_default.cc:31] Failed to open /sys/class/apex: No such file or directory
I driver/usb/local_usb_device.cc:944] EnumerateDevices: vendor:0x1a6e, product:0x89a
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[2] port[0]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[2]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[1]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[4]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[3]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[3]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[2]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[1]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[0]
I driver/usb/local_usb_device.cc:944] EnumerateDevices: vendor:0x18d1, product:0x9302
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[2] port[0]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[2]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[1]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[4]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[3]
I driver/usb/local_usb_device.cc:998] EnumerateDevices: found [/sys/bus/usb/devices/1-1.3.3]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[3]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[2]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[1]
I driver/usb/local_usb_device.cc:979] EnumerateDevices: checking bus[1] port[0]
I driver/beagle/beagle_usb_driver_provider.cc:225] Enumerate: adding path [/sys/bus/usb/devices/1-1.3.3]
I tflite/edgetpu_manager_direct.cc:471] No device of type Apex (PCIe) is available.
I tflite/edgetpu_context_direct.cc:106] USB always DFU: False (default)
I tflite/edgetpu_context_direct.cc:147] USB bulk-in queue capacity: 8
I tflite/edgetpu_context_direct.cc:63] Performance expectation: High when USB connected EdgeTpu is throttled
I driver/usb/usb_driver.cc:1383] Open device and check if DFU is needed
I driver/usb/local_usb_device.cc:1013] OpenDevice: [/sys/bus/usb/devices/1-1.3.3]
I driver/usb/local_usb_device.cc:1050] OpenDevice: checking bus[2] port[0]
I driver/usb/local_usb_device.cc:1050] OpenDevice: checking bus[1] port[2]
I driver/usb/local_usb_device.cc:1050] OpenDevice: checking bus[1] port[1]
I driver/usb/local_usb_device.cc:1050] OpenDevice: checking bus[1] port[4]
I driver/usb/local_usb_device.cc:1050] OpenDevice: checking bus[1] port[3]
I driver/usb/local_usb_device.cc:1081] OpenDevice: device opened 0x55b429dd60
I driver/usb/local_usb_device.cc:184] LocalUsbDevice
I driver/usb/usb_standard_commands.cc:36] UsbStandardCommands
I driver/usb/usb_dfu_commands.cc:37] UsbDfuCommands
I driver/usb/usb_standard_commands.cc:43] GetDeviceDescriptor
I driver/usb/local_usb_device.cc:398] GetDescriptor
I driver/usb/usb_standard_commands.cc:78] Vender ID: 0x18d1
I driver/usb/usb_standard_commands.cc:79] Product ID: 0x9302
I driver/usb/usb_driver.cc:1410] Device is already in application mode, skipping DFU
I driver/usb/usb_driver.cc:1422] Resetting device
I driver/usb/local_usb_device.cc:243] Close: closing device 0x55b429dd60 
I driver/usb/local_usb_device.cc:216] DoCancelAllTransfers: cancelling 0 async transfers
I driver/usb/local_usb_device.cc:224] DoCancelAllTransfers: waiting for all async transfers to complete
I driver/usb/local_usb_device.cc:234] DoCancelAllTransfers: all async transfers have completed
I driver/usb/local_usb_device.cc:276] Close: releasing 0 transfer buffers
I driver/usb/local_usb_device.cc:289] Close: performing graceful reset
I driver/usb/local_usb_device.cc:322] Close: final clean up completed
I driver/usb/usb_driver.cc:1364] Opening device expecting application mode
I driver/usb/local_usb_device.cc:1013] OpenDevice: [/sys/bus/usb/devices/1-1.3.3]
I driver/usb/local_usb_device.cc:1050] OpenDevice: checking bus[2] port[0]
I driver/usb/local_usb_device.cc:1050] OpenDevice: checking bus[1] port[2]
I driver/usb/local_usb_device.cc:1050] OpenDevice: checking bus[1] port[1]
I driver/usb/local_usb_device.cc:1050] OpenDevice: checking bus[1] port[4]
I driver/usb/local_usb_device.cc:1050] OpenDevice: checking bus[1] port[3]
I driver/usb/local_usb_device.cc:1081] OpenDevice: device opened 0x55b42ba410
I driver/usb/local_usb_device.cc:184] LocalUsbDevice
I driver/usb/usb_standard_commands.cc:36] UsbStandardCommands
I driver/usb/usb_ml_commands.cc:47] UsbMlCommands
I driver/usb/usb_dfu_commands.cc:40] ~UsbDfuCommands
I driver/usb/usb_standard_commands.cc:39] ~UsbStandardCommands
I driver/usb/local_usb_device.cc:196] ~LocalUsbDevice
I driver/usb/local_usb_device.cc:243] Close: closing device (nil) 
I driver/usb/local_usb_device.cc:352] ClaimInterface
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a30c
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A30C] == 0xF0059
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A30C] := 0xF0059
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a314
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A314] == 0x110000
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a318
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A318] == 0x50C50258
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a318
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A318] == 0x50C50258
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A318] := 0x50850258
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a318
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A318] == 0x50850000
I driver/usb/usb_ml_commands.cc:117] ReadRegister64 offset 0x44018
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:147] ReadRegister64 [0x44018] == 0x0
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4A000] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x48788] := 0x7F
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:117] ReadRegister64 offset 0x48788
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:147] ReadRegister64 [0x48788] == 0x7F
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40020] := 0x1E02
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a314
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A314] == 0x110000
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A314] := 0x150000
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a000
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A000] == 0x219089A
I driver/usb/usb_driver.cc:321] e-fuse programming revision: 2
I driver/usb/usb_driver.cc:328] InitializeChip Enabling only sc host interrupt descriptors
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C148] := 0xF0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_driver.cc:341] InitializeChip Enabling single EP mode
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C160] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_driver.cc:355] InitializeChip Setting 256B chunk for USB 2 High Speed
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C058] := 0x20
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x44018] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x44158] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x44198] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x441D8] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x44218] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x48788] := 0x7F
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:117] ReadRegister64 offset 0x48788
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:147] ReadRegister64 [0x48788] == 0x7F
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x400C0] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40150] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40110] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40250] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40298] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x402E0] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40328] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40190] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x401D0] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40210] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C060] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C070] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C080] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C090] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C0A0] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a0d4
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A0D4] == 0x1
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A0D4] := 0x80000001
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a704
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A704] == 0x70007F
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A704] := 0x7F
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a33c
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A33C] == 0xC003F
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A33C] := 0x3F
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A500] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A600] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A558] := 0x3
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A658] := 0x3
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a0d8
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A0D8] == 0x0
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A0D8] := 0x80000000
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_driver.cc:1575] Reducing bulk-in request size to 256 bytes for USB2
I tflite/edgetpu_context_direct.cc:174] Opening device at /sys/bus/usb/devices/1-1.3.3
I driver/usb/usb_driver.cc:1155] WorkerThreadFunc starting worker thread
I driver/usb/usb_driver.cc:1174] WorkerThreadFunc dispatching 0 callback events in worker thread
I driver/usb/usb_driver.cc:1210] WorkerThreadFunc Re-installing event reader
I driver/usb/local_usb_device.cc:748] AsyncBulkInTransfer
I driver/usb/local_usb_device.cc:761] ASYNC IN 2 begin
I driver/usb/usb_driver.cc:1232] WorkerThreadFunc Re-installing interrupt reader
I driver/usb/local_usb_device.cc:785] AsyncInterruptInTransfer
I driver/usb/local_usb_device.cc:798] ASYNC IN 3 begin
I driver/usb/usb_driver.cc:1260] WorkerThreadFunc Installing bulk-in reader. buffer index [0]
I driver/usb/local_usb_device.cc:748] AsyncBulkInTransfer
I driver/usb/local_usb_device.cc:761] ASYNC IN 1 begin
I driver/usb/usb_driver.cc:1260] WorkerThreadFunc Installing bulk-in reader. buffer index [1]
I driver/usb/local_usb_device.cc:748] AsyncBulkInTransfer
I driver/usb/local_usb_device.cc:761] ASYNC IN 1 begin
I driver/usb/usb_driver.cc:1260] WorkerThreadFunc Installing bulk-in reader. buffer index [2]
I driver/usb/local_usb_device.cc:748] AsyncBulkInTransfer
I driver/usb/local_usb_device.cc:761] ASYNC IN 1 begin
I driver/usb/usb_driver.cc:1260] WorkerThreadFunc Installing bulk-in reader. buffer index [3]
I driver/usb/local_usb_device.cc:748] AsyncBulkInTransfer
I driver/usb/local_usb_device.cc:761] ASYNC IN 1 begin
I driver/usb/usb_driver.cc:1260] WorkerThreadFunc Installing bulk-in reader. buffer index [4]
I driver/usb/local_usb_device.cc:748] AsyncBulkInTransfer
I driver/usb/local_usb_device.cc:761] ASYNC IN 1 begin
I driver/usb/usb_driver.cc:1260] WorkerThreadFunc Installing bulk-in reader. buffer index [5]
I driver/usb/local_usb_device.cc:748] AsyncBulkInTransfer
I driver/usb/local_usb_device.cc:761] ASYNC IN 1 begin
I driver/usb/usb_driver.cc:1260] WorkerThreadFunc Installing bulk-in reader. buffer index [6]
I driver/usb/local_usb_device.cc:748] AsyncBulkInTransfer
I driver/usb/local_usb_device.cc:761] ASYNC IN 1 begin
I driver/usb/usb_driver.cc:1260] WorkerThreadFunc Installing bulk-in reader. buffer index [7]
I driver/usb/local_usb_device.cc:748] AsyncBulkInTransfer
I driver/usb/local_usb_device.cc:761] ASYNC IN 1 begin
I driver/usb/usb_driver.cc:1317] WorkerThreadFunc waiting on state change
I driver/usb/usb_driver.cc:91] Unlocks both mutex
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
I tflite/edgetpu_manager_direct.cc:226] Releasing Edge TPU device at /sys/bus/usb/devices/1-1.3.3
I tflite/edgetpu_context_direct.cc:180] Closing Edge TPU device at /sys/bus/usb/devices/1-1.3.3
I driver/usb/local_usb_device.cc:216] DoCancelAllTransfers: cancelling 10 async transfers
I driver/usb/usb_driver.cc:86] lock (does nothing)
I driver/usb/usb_driver.cc:1322] WorkerThreadFunc driver state change detected
I driver/usb/local_usb_device.cc:672] ASYNC IN 1 end
I driver/usb/local_usb_device.cc:97] ConvertLibUsbTransferStatus: USB transfer error 3 [LibUsbDataInCallback]
I driver/usb/local_usb_device.cc:642] UnregisterCompletedTransfer
I driver/usb/local_usb_device.cc:224] DoCancelAllTransfers: waiting for all async transfers to complete
I driver/usb/local_usb_device.cc:672] ASYNC IN 1 end
I driver/usb/local_usb_device.cc:97] ConvertLibUsbTransferStatus: USB transfer error 3 [LibUsbDataInCallback]
I driver/usb/local_usb_device.cc:642] UnregisterCompletedTransfer
I driver/usb/local_usb_device.cc:672] ASYNC IN 1 end
I driver/usb/local_usb_device.cc:97] ConvertLibUsbTransferStatus: USB transfer error 3 [LibUsbDataInCallback]
I driver/usb/local_usb_device.cc:642] UnregisterCompletedTransfer
I driver/usb/local_usb_device.cc:672] ASYNC IN 1 end
I driver/usb/local_usb_device.cc:97] ConvertLibUsbTransferStatus: USB transfer error 3 [LibUsbDataInCallback]
I driver/usb/local_usb_device.cc:642] UnregisterCompletedTransfer
I driver/usb/local_usb_device.cc:672] ASYNC IN 1 end
I driver/usb/local_usb_device.cc:97] ConvertLibUsbTransferStatus: USB transfer error 3 [LibUsbDataInCallback]
I driver/usb/local_usb_device.cc:642] UnregisterCompletedTransfer
I driver/usb/local_usb_device.cc:672] ASYNC IN 1 end
I driver/usb/local_usb_device.cc:97] ConvertLibUsbTransferStatus: USB transfer error 3 [LibUsbDataInCallback]
I driver/usb/local_usb_device.cc:642] UnregisterCompletedTransfer
I driver/usb/local_usb_device.cc:672] ASYNC IN 1 end
I driver/usb/local_usb_device.cc:97] ConvertLibUsbTransferStatus: USB transfer error 3 [LibUsbDataInCallback]
I driver/usb/local_usb_device.cc:642] UnregisterCompletedTransfer
I driver/usb/local_usb_device.cc:672] ASYNC IN 3 end
I driver/usb/local_usb_device.cc:97] ConvertLibUsbTransferStatus: USB transfer error 3 [LibUsbDataInCallback]
I driver/usb/local_usb_device.cc:642] UnregisterCompletedTransfer
I driver/usb/local_usb_device.cc:672] ASYNC IN 1 end
I driver/usb/local_usb_device.cc:97] ConvertLibUsbTransferStatus: USB transfer error 3 [LibUsbDataInCallback]
I driver/usb/local_usb_device.cc:642] UnregisterCompletedTransfer
I driver/usb/local_usb_device.cc:672] ASYNC IN 2 end
I driver/usb/local_usb_device.cc:97] ConvertLibUsbTransferStatus: USB transfer error 3 [LibUsbDataInCallback]
I driver/usb/local_usb_device.cc:642] UnregisterCompletedTransfer
I driver/usb/local_usb_device.cc:234] DoCancelAllTransfers: all async transfers have completed
I driver/usb/usb_driver.cc:1174] WorkerThreadFunc dispatching 10 callback events in worker thread
I driver/usb/usb_driver.cc:466] HandleInterrupt cancelled, ignore.
I driver/usb/usb_driver.cc:404] HandleEvent cancelled, ignore.
I driver/usb/usb_driver.cc:1194] All bulk-in buffers are available
I driver/usb/usb_driver.cc:1201] Driver is closing, and all async operations have completed.
I driver/usb/usb_driver.cc:1330] WorkerThreadFunc leaving worker thread
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C070] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C080] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C090] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C0A0] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a0d4
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A0D4] == 0x80000001
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A0D4] := 0x1
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a704
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A704] == 0x7F
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A704] := 0x70007F
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a33c
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A33C] == 0x3F
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A33C] := 0xC003F
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A500] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A600] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A558] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A658] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a0d8
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A0D8] == 0x80000000
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A0D8] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x4C060] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x44018] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x44158] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x44198] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x441D8] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x44218] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x48788] := 0x7F
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:117] ReadRegister64 offset 0x48788
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:147] ReadRegister64 [0x48788] == 0x7F
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x400C0] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40150] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40110] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40250] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40298] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x402E0] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40328] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40190] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x401D0] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:178] WriteRegister64 [0x40210] := 0x2
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a318
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A318] == 0x50850008
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1A318] := 0x50C50008
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:80] ReadRegister32 offset 0x1a318
I driver/usb/local_usb_device.cc:514] SendControlCommandWithDataIn
I driver/usb/local_usb_device.cc:521] SYNC CTRL WITH DATA IN begin
I driver/usb/local_usb_device.cc:538] SYNC CTRL WITH DATA IN end
I driver/usb/usb_ml_commands.cc:110] ReadRegister32 [0x1A318] == 0x50C50258
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1907C] := 0xF
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:153] WriteRegister32 [0x1907C] := 0x0
I driver/usb/local_usb_device.cc:475] SendControlCommandWithDataOut
I driver/usb/local_usb_device.cc:482] SYNC CTRL WITH DATA OUT begin
I driver/usb/local_usb_device.cc:499] SYNC CTRL WITH DATA OUT end
I driver/usb/usb_ml_commands.cc:50] ~UsbMlCommands
I driver/usb/usb_standard_commands.cc:39] ~UsbStandardCommands
I driver/usb/local_usb_device.cc:196] ~LocalUsbDevice
I driver/usb/local_usb_device.cc:243] Close: closing device 0x55b42ba410 
I driver/usb/local_usb_device.cc:263] Close: releasing claimed interface 0
I driver/usb/local_usb_device.cc:216] DoCancelAllTransfers: cancelling 0 async transfers
I driver/usb/local_usb_device.cc:224] DoCancelAllTransfers: waiting for all async transfers to complete
I driver/usb/local_usb_device.cc:234] DoCancelAllTransfers: all async transfers have completed
I driver/usb/local_usb_device.cc:276] Close: releasing 0 transfer buffers
I driver/usb/local_usb_device.cc:322] Close: final clean up completed
37329476
0.001
```

**Comment by tilakrayal** [comment]
@damjandakic93,
Could you please check and confirm if you are facing the same issue with the latest tensorflow versions 2.15 and 2.16?
Thank you!

**Comment by damjandakic93** [comment]
I actually made a mistake, I ran the edgetpu-compiler but copied the wrong file (so no ops were intended for TPU).
I have another issue where I get segfault upon calling Invoke, I guess I'll open a new Issue for that?

**Comment by tilakrayal** [comment]
@damjandakic93,
In that scenario, could you please feel free to move this issue to closed status? Thank you!

**Comment by damjandakic93** [comment]
Agreed, thanks.
